contract-ui/tabs/overview/components/ShareContract.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  useDashboardEVMChainId,
+  useDashboardNetwork,
+} from "../../../../@3rdweb-sdk/react";
+import {
+  AUDITED_BADGE_HEIGHT,
+  BADGE_HEIGHT,
+  BADGE_WIDTH,
+} from "../../../../constants/badge-size";
+import { useTrack } from "../../../../hooks/analytics/useTrack";
```

Comment:
No need to use relative paths, you can just use `@3rdweb-sdk/react`, `constants/badge-size` and `hooks/analytics/useTrack`